magicmaze.js

```diff
@@ -869,26 +902,38 @@ function (dojo, declare) {
         }
         const base = `#mm_control${tokenId} `
         dojo.connect(document.querySelector(base + '> .mm_actionN'), 'onclick', this, function (evt) {
+          // reset possible query after ambiguous action
+          this.updatePageTitle()
```

Comment:
should these be moved to dispatchMove? seems error-prone to repeat this in so many places